trieste/acquisition/rule.py

```diff
@@ -972,6 +1019,107 @@ def update(
         """
         ...
 
+    def _get_tags(self, tags: Set[Tag]) -> Tuple[Set[Tag], Set[Tag]]:
```

Comment:
Would this be better placed in utils/misc?

Reply:
I think it is quite specific to the current use case, i.e. it returns two list containing only global parts of the tags. I propose we leave it here for now, but move it in the future if we find another use case for it.